### PR TITLE
Fix Partners and Sponsors alignment

### LINKED ISSUE #159: Partners and Sponsors is not aligned properly
### Description
Partners and Sponsors are not aligned properly.

### Screenshots/testimonials
issue

### PR DESCRIPTION
**Pull request by SaravgiYash**
### Solves Issue #159 
Description: Partners and Sponsors section aligned properly.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

**Review by tusharjain0022 — APPROVED** [review]
LGTM! 💯

**Comment by hariom1625** [comment]
LGTM !! 
@Yashs911 Since the max-width of the media query is being decreased so for keeping the design optimized, also decrease the ```partners-logo``` border-radius to 30px.

File: client/src/components/Home/PartnersSection/PartnersSection.css [hunk 1]
```diff
@@ -125,14 +125,14 @@ text-decoration: none;
...
 }
 
-@media(min-width:1600px) {
+@media(min-width:1400px) {
       .container.partners-section {
             max-width: 85vw;
       }
 
       .partners-logo, .logo-name {
             width: 25vw;
```
Comment (hariom1625) [inline client/src/components/Home/PartnersSection/PartnersSection.css:134]: A bit of minor change and you are good to go, on hovering the sponsor's name is touching the edges which doesn't look good, so make the logo-name width:23vw or any optimum value which appears good to you.
(For reference hover over IEEE logo).

**Review by hariom1625 — CHANGES_REQUESTED** [review]

File: client/src/components/Home/PartnersSection/PartnersSection.css [hunk 2]
```diff
@@ -125,14 +125,14 @@ text-decoration: none;
...
-            border-radius: 35px;
+            border-radius: 30px;
       }
 
       .logo-name {
```
Comment (hariom1625) [inline client/src/components/Home/PartnersSection/PartnersSection.css:138]: On hovering over the IEEE logo the ```logo-name``` is touching the edges which don't look good. I would suggest you can increase the width to 23vw or any other value which fits the best.

**Review by hariom1625 — CHANGES_REQUESTED** [review]

File: client/src/components/Home/PartnersSection/PartnersSection.css (hunk 2 above)
Comment (SaravgiYash) [reply client/src/components/Home/PartnersSection/PartnersSection.css:138]: @hariom1625 changed the width to 23vw

**Review by hariom1625 — APPROVED** [review]

**Pull request merged**